dblp-aliases.csv

```diff
@@ -78422,3 +78422,4 @@ Liang-liang Wang,Liangliang Wang
 Liang-Liang Wang,Liangliang Wang
 K. Vengattaramane,Kameswaran Vengattaramane
 Peter Wai-Tat Tse,Peter W. Tse
+Peter A. Jansen,Peter Jansen 0001
```

Comment:
Please go ahead and remove this entry - this file is now automatically generated.

Reply:
You do need to enter both aliases in the appropriate csrankings- file.

Reply:
I cannot merge until this file is reverted.

Reply:
Sorry, just took me a few minutes to remember where I'd forked this to change the PR. :)